test/BaseLightStepTest.php

```diff
@@ -8,6 +8,14 @@
  */
 abstract class BaseLightStepTest extends PHPUnit_Framework_TestCase
 {
+    protected function createTestTracer($component_name, $access_token) {
+        $opts = [
+            "debug_disable_flush" => "true"
```

Comment:
Should this instead be `TRUE`, as it is [here](https://github.com/lightstep/lightstep-tracer-php/blob/453e8b8e203476b2491289695bf5d961096fc44e/test/RuntimeTest.php#L7)?

(Excuse my extreme PHP ignorance!)

Reply:
They all evaluate to the same, but happy to align for consistency.